pkg/chartverifier/checks/checks.go

```diff
@@ -337,3 +361,40 @@ func parseImageReference(image string) pyxis.ImageReference {
 	return imageRef
 
 }
+
+func getOCPRange(kubeVersionRange string) string {
+
+	semverCompare := sprig.GenericFuncMap()["semverCompare"].(func(string, string) (bool, error))
+	minOCPVersion := ""
+	maxOCPVersion := ""
+	for kubeVersion, OCPVersion := range tool.GetKubeOpenShiftVersionMap() {
+		match, err := semverCompare(kubeVersionRange, kubeVersion)
+		if err != nil {
+			return fmt.Sprintf("%s : %s", KuberVersionProcessingError, err)
```

Comment:
Nit: can we use the common style for error handling in golang: https://golang.org/doc/effective_go#multiple-returns? i.e. return result as well as the error if any (`func getOCPRange(kubeVersionRange string) string error`).

Reply:
Fixed